lib/ansible/modules/cloud/amazon/ec2.py

```diff
@@ -1687,13 +1711,14 @@ def main():
         if not module.params.get('image'):
             module.fail_json(msg='image parameter is required for new instance')
 
-        if module.params.get('exact_count') is None:
+        if module.params.get('exact_count') is None and \
```

Comment:
I'd somewhat prefer not to because while it's terser, IMO it's harder to read what's going on in.